README.md

```diff
@@ -1,42 +1,17 @@
-# Mapbox GL Shaders
+Dear `mapbox-gl-shaders` contributors,
 
-This repository contains GL shaders which are shared by [Mapbox GL JS](https://github.com/mapbox/mapbox-gl-js) and [Mapbox GL Native](https://github.com/mapbox/mapbox-gl-native).
+Thank you all for being such loyal supporters and champions of the `mapbox-gl-shaders` project. You helped write some amazing cross-platform GLSL.
```

Comment:
It has been an amazing ride, and we’re humbled by the opportunity to be part of something larger – something monolithic.